### PR TITLE
OCD-3910 UI enhancements

### PR DESCRIPTION
**Pull request by mattystank**

### CODE REVIEW

File: src/app/components/listing/details/ics-family/ics-family.jsx [hunk 1]
```diff
@@ -140,137 +170,152 @@ function ChplIcsFamily(props) {
...
+            <CardContent>
+              <figure>
+                <div className={classes.cardContainer}>
+                  <div className={classes.directionContainer}>
+                    <Typography> Select a listing to the right to view more information. You can also click and drag to scroll through the listings.</Typography>
+                    <Typography>
+                      To compare all listings together, please select
+                      {' '}
+                      {' '}
+                      <Link
```
Comment (andlar) [inline src/app/components/listing/details/ics-family/ics-family.jsx]: This should be a ChplLink

File: src/app/components/listing/details/ics-family/ics-family.jsx [hunk 2]
```diff
@@ -140,137 +170,152 @@ function ChplIcsFamily(props) {
...
+                        Close Details
+                      </Button>
+                    </div>
+                  )}
+                  </div>
+                  <CytoscapeComponent
+                    elements={elements}
+                    style={{ width: '50%', height: '40vh' }}
+                    minZoom={1}
+                    maxZoom={1}
```
Comment (andlar) [inline src/app/components/listing/details/ics-family/ics-family.jsx]: Why did these change? There was a lot of work that went into getting them right before. Are these changes necessary, and have you compared the change in zoom on a few different trees of ICS?

File: src/app/components/listing/details/ics-family/ics-family.jsx (hunk 1 above)
Comment (mattystank) [reply src/app/components/listing/details/ics-family/ics-family.jsx]: If I use the ChplLink then we can not alter the flex that causing the link to jump down a line

File: src/app/components/listing/details/ics-family/ics-family.jsx (hunk 2 above)
Comment (mattystank) [reply src/app/components/listing/details/ics-family/ics-family.jsx]: I took out the zoom levels so I user wouldn't get loss within the chart. It very easy to scroll with a mouse and lose your position. We can add them back but I think the functionally isnt helpful

File: src/app/components/listing/details/ics-family/ics-family.jsx (hunk 1 above)
Comment (andlar) [reply src/app/components/listing/details/ics-family/ics-family.jsx]: The problem is that without the "ChplLink" component we don't have a way of tracking analytic events on clicks. We can track navigation, but we won't be able to have an event that says, for instance, "compared from ICS Relationship figure"

File: src/app/components/listing/details/ics-family/ics-family.jsx (hunk 2 above)
Comment (andlar) [reply src/app/components/listing/details/ics-family/ics-family.jsx]: So you disabled the zoom? If we want to do that, that's fine, but then we should do it with the actual "disable zoom" configuration. Have you looked at a Listing with a lot of inheritance? I found being able to zoom out to see all of the items, even if you couldn't read the text, was sometimes useful. I also don't really want to remove functionality that's been out there for a while without getting confirmation that we should
